### PR TITLE
Notify management

### PR DESCRIPTION
**Pull request by sjxqqq**

### CODE REVIEW

File: cmd/gzv/cli/update/update.go [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+	"time"
+)
+
+const OldVersion = false
+const NewVersion = true
+const UpdatePath = "update"
+const System = runtime.GOOS
+const CheckVersionGap = time.Hour
+const Timeout = time.Second * 60
+const DefaultRequestURL = "http://47.110.159.248:8000/request"
```
Comment (bird512) [inline cmd/gzv/cli/update/update.go]: should use a domain instead of IP and should use https

File: cmd/gzv/cli/update/type.go [hunk 2]
```diff
@@ -0,0 +1,27 @@
+package update
+
+type UpdateInfo struct {
+	PackageUrl  string   `json:"package_url"`
+	Packagemd5  string   `json:"package_md5"`
```
Comment (bird512) [inline cmd/gzv/cli/update/type.go]: use PackageMd5 is better

File: cmd/gzv/cli/update/update.go [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+
+import (
+	"fmt"
+	"github.com/zvchain/zvchain/common"
+	"github.com/zvchain/zvchain/log"
+	"runtime"
+	"time"
+)
+
+const OldVersion = false
```
Comment (bird512) [inline cmd/gzv/cli/update/update.go]: consts used in same package do not need to mark as public

File: cmd/gzv/cli/update/update.go [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+		go nm.processOutput(timeOut)
+
+		//Check if the latest version has been downloaded locally
+		if isFileExist(vc.localFileName) {
+			log.DefaultLogger.Errorln("The latest version has been downloaded locally, but not yet run")
+			fmt.Println("The latest version has been downloaded locally, but not yet run")
+			return
+		}
+		log.DefaultLogger.Infoln("start download ...")
+		err := vc.download()
```
Comment (bird512) [inline cmd/gzv/cli/update/update.go:91]: Should skip download if last downloading not finished yet

File: go.mod [hunk 5]
```diff
@@ -21,14 +21,14 @@ require (
 	github.com/howeyc/gopass v0.0.0-20170109162249-bf9dde6d0d2c
 	github.com/kr/pretty v0.1.0 // indirect
 	github.com/minio/sha256-simd v0.1.0
+	github.com/mitchellh/mapstructure v1.1.2
```
Comment (bird512) [inline go.mod]: this package is not used in project

**Review by darren0718 — DISMISSED** [review]

**Review by darren0718 — DISMISSED** [review]

**Review by darren0718 — APPROVED** [review]

**Review by bird512 — APPROVED** [review]

**Pull request merged**